### DISCUSSION TITLE
nuget.org ID prefix reservation for Neuroglia packages

### DISCUSSION THREAD

**Discussion by KevinD-87** [neuroglia-io/AsyncApi#34] [Ideas]
While setting up package source mapping for my project, I noticed the Neuroglia package prefix was not reserved on nuget.org.
This opens up the possibility for a bad actor to basically hijack your packages and publish newer versions with the possibility of adding malware.
You can find more info about this here: https://learn.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation
So I'd like to raise the idea of reserving this prefix for your company to protect your packages against such attacks. There's no cost attached in doing so.

**Comment by cdavernas** [comment]
Many thanks @KevinD-87! I didn't know about that!!!
Process has been initiated as of today, let's see if/when it'll be approved.
